test/integration/data/integration-init-state.json

```diff
@@ -0,0 +1,2042 @@
+        "mobileActiveAndroid": true,
+        "mobileActiveIOS": true,
+        "mobile_active": true,
+        "smartTransactions": {}
+      },
+      "polygon": {
+        "extensionActive": true,
+        "extension_active": true,
+        "fallbackToV1": false,
+        "fallback_to_v1": false,
+        "mobileActive": true,
+        "mobileActiveAndroid": true,
+        "mobileActiveIOS": true,
+        "mobile_active": true,
+        "smartTransactions": {}
+      },
+      "smartTransactions": {
+        "extensionActive": true,
+        "mobileActive": true,
+        "mobileActiveAndroid": false,
+        "mobileActiveIOS": false
+      },
+      "smart_transactions": {
+        "extension_active": true,
+        "mobile_active": true
+      },
+      "swapRedesign": {
+        "extensionActive": true,
+        "mobileActive": false
+      },
+      "zksync": {
+        "extensionActive": true,
+        "extension_active": true,
+        "fallbackToV1": false,
+        "fallback_to_v1": false,
+        "mobileActive": true,
+        "mobileActiveAndroid": true,
+        "mobileActiveIOS": true,
+        "mobile_active": true,
+        "smartTransactions": {}
+      }
+    }
+  },
+  "switchedNetworkNeverShowMessage": false,
+  "termsOfUseLastAgreed": 1692109813199,
+  "theme": "os",
+  "tokenList": {
+    "0x2260fac5e5542a773aa44fbcfedf7c193bc2c599": {
+      "address": "0x2260fac5e5542a773aa44fbcfedf7c193bc2c599",
+      "symbol": "WBTC",
+      "decimals": 8,
+      "name": "Wrapped Bitcoin",
+      "iconUrl": "https://s3.amazonaws.com/airswap-token-images/WBTC.png",
+      "aggregators": [
+        "airswapLight",
+        "bancor",
+        "cmc",
+        "coinGecko",
+        "kleros",
+        "oneInch",
+        "paraswap",
+        "pmm",
+        "totle",
+        "zapper",
+        "zerion",
+        "zeroEx"
+      ],
+      "occurrences": 12
+    },
+    "0x0bc529c00c6401aef6d220be8c6ea1667f6ad93e": {
+      "address": "0x0bc529c00c6401aef6d220be8c6ea1667f6ad93e",
+      "symbol": "YFI",
+      "decimals": 18,
+      "name": "yearn.finance",
+      "iconUrl": "https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/ethereum/assets/0x0bc529c00C6401aEF6D220BE8C6Ea1667F6Ad93e/logo.png",
+      "aggregators": [
+        "airswapLight",
+        "bancor",
+        "cmc",
+        "coinGecko",
+        "kleros",
+        "oneInch",
+        "paraswap",
+        "pmm",
+        "totle",
+        "zapper",
+        "zerion",
+        "zeroEx"
+      ],
+      "occurrences": 12
+    }
+  },
+  "tokens": [
+    {
+      "address": "0x108cf70c7d384c552f42c07c41c0e1e46d77ea0d",
+      "symbol": "TEST",
+      "decimals": "0"
+    },
+    {
+      "address": "0xd8f6a2ffb0fc5952d16c9768b71cfd35b6399aa5",
+      "decimals": "8",
+      "symbol": "TEST2"
+    },
+    {
+      "address": "0x617b3f8050a0bd94b6b1da02b4384ee5b4df13f4",
+      "symbol": "META",
+      "decimals": "18"
+    }
+  ],
+  "transactions": [
```

Comment:
as an init state, do we want to have already several prepopulated tx's, networks etc.. or would we rather have a more "clean" state, and then add the specific data  on each test that needs it?

Reply:
ok so @matthewwalsh0 also hinted at the same. I would rather follow the fixtures approach, but I felt that it would significantly raise complexity for this PR. That being said, I'll come up with a proposal on how to have the fixtures style approach for the integration tests